docs/hooks/useTransactor.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# useTransactor
+
+Use this hook to interact with the chain and get UI feedback on the transaction status. You can pass in anything that is a valid parameter to [Viem's `sendTransaction` function](https://viem.sh/docs/actions/wallet/sendTransaction#parameters).
+
+```ts
+const transactor = useTransactor();
+const writeTx = transactor({
+  to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  value: 1000000000000000000n,
+});
+await writeTx();
+```
+
+This example tries to send 1 ETH to the address `0x70997970c51812dc3a010c7d01b50e0d17dc79c8`, prompting the connected `WalletClient` for a signature. And in the case of a successful transaction, it will show a popup in the UI with the message: "🎉 Transaction completed successfully!".
+
+It's also possible to pass in a promise from [`writeContractAsync` from `useWriteContract`](/hooks/useScaffoldWriteContract) as the first argument. This way, you will also have access to data from the transaction, such as the current status and the result.
+```ts
+const { isPending, writeContractAsync: writeYourContractAsync } = useScaffoldWriteContract("YourContract");
+const transactor = useTransactor();
+
+...
+
+<button
+  className="btn btn-primary"
+  disabled={isPending}
+  onClick={async () => {
+    try {
+      const contractWritePromise = writeYourContractAsync({
+        functionName: "setGreeting",
+        args: ["The value to set"],
+        value: parseEther("0.1"),
+      });
+      await transactor(contractWritePromise);
+    } catch (e) {
+      console.error("Error setting greeting:", e);
+    }
+  }}
+>
+  Set Greeting
+</button>
+```
```

Comment:
Actually there is a mistake in this example we are using scaffold write hook's `writeContractAsync` which already internally uses `useTransactor`. 

Also we already have the [recipe](https://docs.scaffoldeth.io/recipes/WagmiContractWriteWithFeedback) for to demonstrate this example, so maybe we could point people to it directly? 


So maybe we could do: 

It also possible to pass it an promise that resolves in with a transaction hash for example promise from wagmi's `writeContractAsync`.  This will show UI feedback until promise is resolved along with link to corresponding blockexplorer for transaction hash. For detailed example checkout [recipe](recipes/WagmiContractWriteWithFeedback)

^ lol I think the above english can be improved.

Reply:
https://github.com/scaffold-eth/se-2-docs/commit/2741ccdcf0a266bc2d64a3c2c0fa40120e3574f2